addons/website_sale/views/templates.xml

```diff
@@ -1338,12 +1338,8 @@
     </template>
 
     <template id="extra_info_option" name="Extra Step Option" inherit_id="wizard_checkout" active="False">
-        <!-- Add a "flag element" to trigger style variation -->
-        <xpath expr="//div[hasclass('wizard')]/div" position="before">
-            <span class="o_wizard_has_extra_step d-none"/>
-        </xpath>
-        <xpath expr="//div[@id='wizard-step20']" position="after">
-            <a class="no-decoration" t-att-href="step&gt;=30 and '/shop/extra_info' or '#'">
+        <xpath expr="//div[@id='wizard-step20']/parent::a" position="after">
+            <a class="no-decoration col" t-att-href="step&gt;=30 and '/shop/extra_info' or '#'">
```

Comment:
Imo, the CSS update should be in a separated commit. I'm not sure to fully understand why this is needed.
Since it has been approved by @Williambraecky , I trust you on this part

Reply:
I agree on the separate commit part.
Prior to the css change the columns were hardcoded to take 1/3 (or 1/4 if extra step was enabled) instead of using existing classes.

@JulienVR Have you checked that mobile display is still coherent?

Reply:
Okay, I'll do a separate commit. @Williambraecky, I forgot to check, I had to add 
```scss
.col {
        @include media-breakpoint-down(md) {
            flex: none;
        }
    }
```
below `.progress-wizard` in the scss, otherwise, I obtained this: 
![image](https://user-images.githubusercontent.com/60702485/214062049-ca3dca7a-0f2c-4b17-a2b5-37b7ebd6f03b.png) instead of this: 
![image](https://user-images.githubusercontent.com/60702485/214062165-5a3565c2-b760-46ee-9b5a-e526e273f947.png)

Reply:
Thanks :)